MrtCore/winui_desktop_packaged_app/ClassLibrary/ClassLibrary.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <RootNamespace>ClassLibrary</RootNamespace>
+    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Remove="ClassLibraryResources.resw" />
+  </ItemGroup>

```

Comment:
redundant

Reply:
If we're adding the file to PRIResource, it makes sense to remove it from None. It belonging to both at the same time doesn't make sense (thinking about it from a Build Action point of view).

Also, this was automatically done. I think it's part of the WinUI project template. If you want, I can remove it.

Reply:
doesn't include in PRIResource remove it from none automatically? It's strange you have both an include and a remove for the same file.

Reply:
They are separate lists in MSBuild. You have to modify them separately.

Reply:
what? so a file is in "none" by default unless you explicitly remove it? That's strange. Also does that mean an existing project with PRIResource, and upgrade to Reunion with your build action fix will be broken?

Reply:
All files aren't in None by default. Some might be in Content by default, some in PRIResource by default, etc., depending on the project type.

With my fix, I should have removed the files we added to PRIResource/Content from None. I don't think that affects any functionality, but it's still the right thing to do. I can do that soon.

Reply:
so you mean resw is in None by default? With your fix, If a project has "<PRIResource Include="a.resw" />", will a.resw be added to PRIResource twice?

Reply:
If you copy paste a RESW file into your project folder and add it to the project, it'll belong to the None list.
If you create a new RESW file, it'll belong to the PRIResource list because of the WinUI project template.
With my fix, we only add RESW files from None to PRIResource, so we won't ever have the same RESW file twice in PRIResource.

Reply:
TBH, i don't understand why we need to do remove here.

Reply:
My understanding is that if a file belongs to None, it means that it doesn't belong to any defined group (PRIResource, Content, etc.) So, a file belonging to None and a non-None list at the same time doesn't make sense.

Reply:
does line 16 make it belong to PRIResource?

Reply:
Yes. My point is that that's not sufficient. In MSBuild, None and PRIResource are two separate lists. You have to modify them separately.

Reply:
It turns out your fix already duplicates the file in both lists, and it doesn't seem harmful. I think we shouldn't burden the developer to add this unusual remove.

Reply:
Talked offline. Developer doesn't need to do anything. The include and remove lines are added automatically when they create a new RESW file in a WinUI project. Starting with version 0.8, including preview, all RESW cases will be handled automatically due to this change: https://github.com/microsoft/ProjectReunion/pull/771.